src/System.Devices.Gpio/System.Devices.Gpio/UnixDriver.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System.Collections.Generic;
```

Comment:
Missing header

// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.